jekyll/_cci2/browser-testing.md

```diff
@@ -97,16 +97,16 @@ CircleCI has developed a Sauce labs browser testing orb that enables you to open
 ```
 version: 2.1
 orbs:
-  saucelabs: saucelabs/connect@volatile
+  sauce-connect: saucelabs/connect@1.0.1
```

Comment:
this GH issue says it should be `sauce-connect: saucelabs/sauce-connect@1.0.1`

Reply:
Yeah...missed one:(.

Fixed